src/pat/upload/upload.js

```diff
@@ -61,6 +61,11 @@ export default Base.extend({
 
         var self = this;
 
+        if (self.$el.find(".upload-area").length) {
+            console.warn("Found already a dropzone on element, skipping!");
```

Comment:
I'd use the info level, otherwise people might get scared of a warning which is harmless.
Also, please use the logging infrastructure. I'd change it to:

```
import logger from "@patternslib/patternslib/src/core/logging";

const log = logger.getLogger("pat-upload");

[...]

        if (self.$el.find(".upload-area").length) {
            logger.info("Found already a dropzone on element, skipping!");
```

Reply:
thanks, done